qtop.py

```diff
@@ -285,7 +284,12 @@ def calculate_job_counts(user_names, job_states):
     expand_useraccounts_symbols(config, user_names)
     state_abbrevs = config['state_abbreviations'][scheduler]
 
-    job_counts = create_job_counts(user_names, job_states, state_abbrevs)
+    try:
+        job_counts = create_job_counts(user_names, job_states, state_abbrevs)
+    except JobNotFound as e:
+        logging.critical('Job state %s not found. You may wish to add '
+                         'that node state inside %s in state_abbreviations section.\n' % (e.job_state, QTOPCONF_YAML))
+
```

Comment:
touches issue #98 
In general I'm trying to add more raises in called functions, and more catches in caller functions, it's the pythonic thing to do!